README.md

```diff
@@ -1,4 +1,34 @@
-# Sortable
+# Admin Sortable
+Admin Sortable is a Laravel package for Craftable used to add sortable feature for you Eloquent models.
+
+## Requirements
+Admin Sortable requires:
+1. php 7.1+
+
+## Installation
+
+### Steps
+
+1. Run `composer require brackets/admin-sortable` in your Craftable project folder.
+2. Run `npm install @shopify/draggable@1.0.0-beta.8` in your Craftable project folder.
```

Comment:
tu by som dal prec tu verziu, podla mna to je zbytocne, ked to je lastest verzia aj tak